### ISSUE TITLE
[BUG] Being buried with a marker doesn't send you to the lobby anymore, whether you are in the underworld or not.

### ISSUE THREAD

**Issue by TonyKarsh** [Blackstone-SS13/BLACKSTONE#986]
Issue reported from  Round ID: 392 (Blackstone (Rougetown on a Dedicated Server))

Reporting client version: 515.1636

I can't say for the underworld part, but being buried with a grave marker, whether or not you have left your body, will not send you to the lobby. I didn't see anywhere whether this was intended or not, but it means that there's no point to burying bodies with a marker, since people will have to journey to the underworld anyway. I have never once been sent to lobby from the underworld, and I'm pretty sure I have been buried while in the underworld before.

For context, in this round I was Red the orphan, got beheaded and rotted, and left my body to observe as a ghost, but didn't leave to the underworld. I watched my body and head get buried near the bog town shylock and stockpile and saw them put up the cross and got the message: "Your soul finds peace, buried in creation, thanks to <I forgot the name>." I remained a ghost, didn't get sent to lobby, and had to journey to the underworld.

**Comment by CVSthePharmacy** [comment]
I can confirm this, I haven't been sent to the lobby the recent times I've been buried

**Comment by Wintermote** [comment]
This has also been my experience; I have been buried multiple times and all it has ever done is send me to the underworld, never to lobby.

Likewise, something like 7/10 people I bury and give burial rites to, will not have successful rites.

The whole thing is very opaque and it really doesn't seem like it's actually working properly.

**Comment by Wintermote** [comment]
Adding to this further, I stayed in my corpse while a gravedigger dragged me to the cemetary, then just as they started putting my body into the pit I went down to the underworld. 

Presumably they finished the burial and did the rites/gravemarker, but I didn't get kicked to lobby.

I'm 99% sure at this point that burial and burial rites don't actually do anything. Possibly the only thing that works is burying with a coin to give a toll.

**Comment by ChungusGamer666** [comment]
Will look into this. Is it really fucked?

**Comment by EnzoSeptum** [comment]
Yes, I was spectating my body, the grave guy did proper ritual I even got a message but was not returned to the lobby or anything.

**Comment by TonyKarsh** [comment]
Round ID: 547 (Blackstone (Rougetown on a Dedicated Server))

I just was able to bury someone with a cross who recently died, and upon tying the cross I got this weird error message. Possibly related to why people aren't going back to lobby screen after proper burial?

![dreamseeker_EDyfDPs8es](https://github.com/Blackstone-SS13/BLACKSTONE/assets/93961316/16651407-071b-4e22-85f0-3d8018378e6e)

**Comment by pilotmans** [comment]
Yeah I've also experienced the exact same things, it also seems like morticians aren't able to use their burial rite spell either.
